.gitsubprojects

```diff
@@ -3,5 +3,4 @@ git_subproject(vmmlib https://github.com/Eyescale/vmmlib.git 639ccfd)
 git_subproject(zeq https://github.com/HBPVis/zeq.git a690304)
 git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git 7d3b9b2)
 git_subproject(dash https://github.com/BlueBrain/dash.git 6e05142)
-git_subproject(Collage https://github.com/Eyescale/Collage.git f615b9c)
```

Comment:
I've had a discussion with @rdumusc in another PR. This is potentially dangerous, as your force another Collage version then the one used by Equalizer (different ABIs). We should have a broader discussion, but a first shot was to only use subprojects which are declared as dependent (leaky) dependencies.